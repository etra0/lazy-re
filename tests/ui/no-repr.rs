use lazy_re::{lazy_re, LazyRe};

#[derive(LazyRe)]
#[lazy_re]
struct Baz {
    no_offset: usize,

    #[offset = 0x42]
    offset: usize,
}

fn main() {}
