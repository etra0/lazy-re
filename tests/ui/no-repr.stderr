error: The struct does not have the attribute #[repr(C, packed)]
 --> tests/ui/no-repr.rs:4:8
  |
4 | struct Baz {
  |        ^^^
